samples/client_query_w_timestamp_params.py

```diff
@@ -32,7 +32,9 @@ def client_query_w_timestamp_params() -> None:
             )
         ]
     )
-    query_job = client.query(query, job_config=job_config)  # Make an API request.
+    results = client.query_and_wait(
+        query, job_config=job_config
+    )  # Make an API request.
 
     for row in query_job:
```

Comment:
Please rename this line too.

```suggestion
    for row in results:
```

Reply:
Yes, I will rename this one.